contracts/modules/capital/Pool.sol

```diff
@@ -461,17 +462,6 @@ contract Pool is IPool, MasterAware, ReentrancyGuard {
     return true;
   }
 
-
-  /// @dev DEPRECATED, use calculateNXMForEth function instead! Returns the amount of wei a seller
-  /// will get for selling NXM
-  ///
-  /// @param amount     Amount of NXM to sell
-  /// @return weiToPay  Amount of wei the seller will get
-  /// [todo] Is it safe to remove this?
-  function getWei(uint amount) external view returns (uint weiToPay) {
```

Comment:
I think this was used by Armor / some distributor contract, and that's why we kept it. Pls double check before removing it.

Reply:
It's used by the yInsure contract

https://github.com/EaseDeFi/arNFT/blob/master/contracts/externals/yInsure.sol#L580

https://etherscan.io/address/0x181aea6936b407514ebfc0754a37704eb8d98f91#code

The issue is even though in theory removing this would brick the sellNXMTokens function, in practice the function is not callable anyway because MCR is at 100% so you can't sell.

The way, to pull out the NXM out of that contract is to run nxmTokenApprove
And transfer them out by the owner (https://etherscan.io/address/0x2D407dDb06311396fE14D4b49da5F0471447d45C) who is Andre i believe.